src/aks-preview/azext_aks_preview/decorator.py

```diff
@@ -2076,11 +2125,11 @@ def get_apiserver_subnet_id(self) -> Union[str, None]:
 class AKSPreviewCreateDecorator(AKSCreateDecorator):
```

Comment:
No update code path was implemented?

Reply:
As discussed offline, we don't need ```aks update``` implemented for this change